CMakeLists.txt

```diff
@@ -18,7 +18,10 @@ mark_as_advanced(FORCE RR_USE_VULKAN)
 
 #global settings
 if (WIN32)
-    add_definitions(/MP -D_CRT_SECURE_NO_WARNINGS)
+    add_definitions(/MP -D_CRT_SECURE_NO_WARNINGS /Zc:preprocessor)
+    if (CMAKE_GENERATOR_PLATFORM STREQUAL "arm64")
+        set (CMAKE_LIBRARY_ARCHITECTURE arm64)
+    endif (CMAKE_GENERATOR_PLATFORM STREQUAL "arm64")
```

Comment:
Do I need to port Embree? [(Line 49)](https://github.com/Unity-Technologies/RadeonRays_SDK/pull/6/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR49)
It doesn't fail without it...

Reply:
No, we aren't using the Embree code path